CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+### [v2.11 _(May 08, 2020)_](https://github.com/omise/omise-magento/releases/tag/v2.11)
```

Comment:
Ideally, we go from down header levels one-by-one, so `#` (H1) to `##` (H2) to `###` (H3).  This skips directly from H1 to H3.